tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -275,6 +283,21 @@ void ral_gemm(ExecutionContext* ctx, void* stream_handle, MemRefType<InT, 2> A,
     TAO_VLOG(1) << "ral_gemm: early return for empty tensor";
     return;
   }
+  if (std::is_same<InT, Eigen::half>::value || std::is_same<InT, float>::value) {
+    auto gpu_driver = ctx->getDriver<GPUDriver>(GPUDriver::name());
+    auto stream =
+       static_cast<se::Stream*>(gpu_driver->asSEStream(ctx, stream_handle));
+    void* s = stream->implementation()->GpuStreamHack();
+    bool fp16_in = std::is_same<InT, Eigen::half>::value;
+    bool fp16_out = std::is_same<OutT, Eigen::half>::value;
+    bool ret = blade_gemm(s, fp16_in, fp16_out,
```

Comment:
I tend to name it just as cutlass_gemm, to avoid too much concepts.
I guess "blade" here is confusing to the readers.

Reply:
No, it might not be cutlass gemm.